visual-recognition/src/main/java/com/ibm/watson/visual_recognition/v4/VisualRecognition.java

```diff
@@ -201,12 +190,9 @@ public void setVersion(final String version) {
     builder.query("version", String.valueOf(this.version));
     MultipartBody.Builder multipartBuilder = new MultipartBody.Builder();
     multipartBuilder.setType(MultipartBody.FORM);
-    for (String item : analyzeOptions.collectionIds()) {
```

Comment:
Make sure this is supposed to be removed and not a purposeful hand edit being removed